patches/chrome-installer-mini_installer-appid.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/installer/mini_installer/appid.h b/chrome/installer/mini_installer/appid.h
+index c408242814504aae086fc332f0d8bbd7872821a2..deadf14c9c8a17d7d83fcf3428cd3734a92d70b1 100644
+--- a/chrome/installer/mini_installer/appid.h
++++ b/chrome/installer/mini_installer/appid.h
```

Comment:
why are we modifying this when we already have brave_appid?

Reply:
We don't have copied header file.
Strangely, `brave_appid.cc` is not located in `chromium_src`. We want to move it under `chromium_src` right? also copy header file, too?